vars/stageStartOpenshiftBuild.groovy

```diff
@@ -5,6 +5,11 @@ def call(def context, def buildArgs = [:]) {
       return
     }
 
+    if (! resourceExists("${context.targetProject}", "dc", "${context.componentId}")) {
+      println("CHK: invoking context.tailorUsingOpenshiftFolder()")
```

Comment:
hi! this should go out or be kept as a debut message, right?

Reply:
Good catch! I would remove it.